rviz_simulator/CMakeLists.txt

```diff
@@ -165,22 +185,54 @@ include_directories(include
 ## add_executable(one_marker src/one_marker.cpp)
 ## target_link_libraries(one_marker ${catkin_LIBRARIES})
 
+FIND_PACKAGE(yaml-cpp REQUIRED)
```

Comment:
I'll address all `CMakeList.txt`-related comments in a separate PR **after** milestone 3 is merged, since the `CMakeList.txt` file from milestone 2 is the same file in milestone 3 (which is a branch of 2) and changing the file in one branch and merging with the other may result in  merge conflicts.